Codecademy/gamut
# feat(DropdownList): Allow for dropdown item to open in new tab

### Overview



Add optional target prop to DropdownList to allow a dropdown list item link to open in a new tab.



### PR Checklist

- [x] Related to JIRA ticket: [EGG-2770](https://codecademy.atlassian.net/browse/EGG-2770)

#### Testing Instructions

We will be using this new `target` prop in portal-app right away, but not the monolith. Please follow the testing instructions in the portal-app PR using this behavior, and check the monolith PR to ensure tests are passing.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/32522)  | [Monolith Env](https://pr-32522-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/portal-app/pull/2562)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2562)   |
